tests/test_integration_layout.py

```diff
@@ -22,7 +22,6 @@
 @pytest.mark.integration_test  # this is a slow test, we may want to skip them in some contexts
 @pytest.mark.parametrize(
     (
-        "screen_size",
```

Comment:
I removed this from our test cases' params while I was there - since it doesn't vary we can just hard-code it in the test itself :slightly_smiling_face: